python/dgl/distributed/graph_partition_book.py

```diff
@@ -874,6 +874,18 @@ def __init__(self, part_id, num_parts, node_map, edge_map, ntypes, etypes):
         self._nid_map = IdMap(self._typed_nid_range)
         self._eid_map = IdMap(self._typed_eid_range)
 
+        # Local node/edge type offset that maps the local homogenized node/edge IDs
+        # to local heterogenized node/edge IDs.  One can do the mapping by binary search
+        # on these arrays.
+        self._local_ntype_offset = np.cumsum(
```

Comment:
pls add pubic method to access these data members. let's avoid access directly. `@property` should be ok.